packages/squircle-element-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@squircle-js/react",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
This is done automatically by changesets, please don't change this manually.

Run `pnpm changeset` instead, and describe your change in a minor version.